### DISCUSSION TITLE
installation error

### DISCUSSION THREAD

**Discussion by VIncewww** [bmaltais/kohya_ss#1940] [Q&A]
<img width="846" alt="螢幕擷取畫面 2024-02-07 183858" src="https://github.com/bmaltais/kohya_ss/assets/139464115/f46456f1-654e-47bd-8ebb-223bffc91130">
Hi i just came across this issue, any help would be appreciated

**Comment by kferterb** [comment]
in the requirements.txt file, change the Gradio version to 3.44.0

**Comment by NeoLoger** [comment]
I changed the Gradio version to 3.44.0 and kohya_ss starts now.
However I'm now getting a `torch.cuda.OutOfMemoryError`

I have formatted the PC from windows11 to Ubuntu 22.04 to see if I can get a little boos in performance but the same configs and data sets that worked fine on windows now get out of memory, and I have no idea if its related...

**Reply by kferterb** [reply]
What kind of video card do you have? How much video memory? Are the "fp16" checkboxes enabled?

**Reply by NeoLoger** [reply]
OS: Ubuntu 22.04
NVIDIA GeForce RTX 3060Ti 8GB
And I have 32GB RAM

And yes the fp16 is enabled, I have played with it tired to change it around.
I event tried the `Full fp16 training (experimental)`

This worked without any issues on my Windows11 install, and this is the same config file of a lora that I have already trained on the same dataset.
My concern is that I have issue with some configuration on the OS or python maybe.
I have no idea...
